src/hotspot/share/classfile/symbolTable.cpp

```diff
@@ -592,6 +612,20 @@ void SymbolTable::dump(outputStream* st, bool verbose) {
     if (!_local_table->try_scan(thr, ds)) {
       log_info(symboltable)("dump unavailable at this moment");
     }
+    if (!_shared_table.empty()) {
+      st->print_cr("---------------");
+      st->print_cr("Shared symbols:");
+      st->print_cr("---------------");
+      DumpSharedSymbol dss(st);
+      _shared_table.iterate(&dss);
+    }
+    if (!_dynamic_shared_table.empty()) {
+      st->print_cr("---------------");
```

Comment:
I think it's better to prefix the comments with `#`. That way it will be easier to include the string/symbol output into -XX: SharedArchiveConfigFile.

Reply:
Fixed.

Thanks for your review.